fhir-server/src/main/java/com/ibm/fhir/server/util/FHIRRestHelper.java

```diff
@@ -1324,11 +1324,13 @@ private void methodValidation(HTTPVerb method, Resource resource) throws FHIRPer
                         "Bundle.Entry.resource is required for BundleEntry with PUT method.";
                 throw buildRestException(msg, IssueType.INVALID);
             }
+            /*
```

Comment:
Did you intend to leave this commented out? Should probably remove it.

Reply:
Removed commented out code.